about.html

```diff
@@ -0,0 +1,154 @@
+              Child Mortality, Boost to Maternal Health, Combat HIV/AIDS,
+              Malaria and other Diseases, Environmental Sustainability and
+              Global Partnership in Development.
+            </p>
+          </div>
+
+          <p>
+            contact us: <br />
+            <a href="email:andis@andis.com">andis-email@andis.mail</a>
+          </p>
+        </div>
+      </div>
+
+      <section class="standout container">
+        <div class="wrapper">
+          <img src="./assets/img/icons-3.svg" class="icon" alt="" />
+          <div class="info">
+            <h2 class="title">Anambra integrated development strategy</h2>
+            <p>Promoting Excellence and Securing the Future</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="container">
+        <div id="diff" class="wrapper">
+          <h2 class="section-heading">Roll Back</h2>
+          <p>Discover more events and information below:</p>
+
+          <div class="other-programs">
+            <a
+              href="#"
+              class="program"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img src="./assets/img/background_4.jpg" alt="andis 2020" />
+              <span class="cover"></span>
+              <h3>2019</h3>
+              <p>ANDIS</p>
+            </a>
+            <a
+              href="#"
+              class="program"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img src="./assets/img/background_img3.jpg" alt="andis 2020" />
+              <span class="cover"></span>
+              <h3>2020</h3>
+              <p>ANDIS</p>
+            </a>
+            <a
+              href="#"
+              class="program"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                src="./assets/img/sfd-map.jpg"
+                alt="ANDIS Official Website"
+              />
+              <span class="cover"></span>
+              <h3>MAP</h3>
+              <p>Official website</p>
+            </a>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <section id="sponsors">
+      <h2 class="section-heading">Sponsors</h2>
+      <div class="sponsors-list">
+        <img
+          src="./assets/img/sponsor1.png"
+          title="sabmiller"
+          alt="sabmiller"
+        />
+        <img src="./assets/img/airpeace.png" title="airpeace" alt="airpeace" />
+        <img src="./assets/img/sponsor3.png" title="dangote" alt="dangote" />
+      </div>
+    </section>
+
+    <footer class="container">
+      <div class="wrapper">
+        <div class="branding">Anambra<br />State<br />Town Hall Meeting</div>
+        <div class="info">
+          <p class="copyright">
+            &copy; 2022 ANDIS<br />Some Rights Reserved<br />CC BY-SA 4.0
+          </p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Kindly remove the `sponsors` section from the about page it doesn't exist on the original design.
- [ ] Kindly change the background on the footer section on the about page to match the original design.
- [ ] Kindly on the footer section make the logo of your page on the left and the text content on the right on both the home and about pages to match the layout of the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/169189593-1610ab9b-5100-4bcc-bff7-da19299b0e80.png)


Original design
![image](https://user-images.githubusercontent.com/35535130/169189288-923f2d7a-d56a-445d-ba33-71d3a72b9d5a.png)